frontend/src/components/worksheets/BundleDetail/BundleActions.jsx

```diff
@@ -27,6 +27,20 @@ class BundleActions extends React.Component<
 			});
 		});
 		run.dependencies = dependencies;
+		run.disk = bundleInfo.metadata.request_disk;
+		run.cpu = bundleInfo.metadata.request_cpus;
+		run.gpu = bundleInfo.metadata.request_gpus;
+		run.memory = bundleInfo.metadata.request_memory;
+		let docker_image = bundleInfo.metadata.docker_image;
+		if (docker_image && docker_image.startsWith('codalab/default-cpu')){
```

Comment:
Why is this logic needed?

Reply:
Otherwsie it shows codalab/default-cpu@sha* for defaults

Reply:
Would be good to document.  But wouldn't the sha* show up for other docker images too?  We could use `request_docker_image`, which doesn't have the sha.

Reply:
Ah true, I will change to use that